js/background.js

```diff
@@ -893,26 +933,26 @@ if (!SAFARI) {
 
             addMenu(translate("block_this_ad"), function(tab, clickdata) {
                 emit_page_broadcast(
-                    {fn:'top_open_blacklist_ui', options:{info: clickdata}},
-                    {tab: tab}
+                    { fn:"top_open_blacklist_ui", options: { info: clickdata } },
+                    { tab: tab }
```

Comment:
@kpeckett We should use spaces in the code like in the following example:
```foo: { bar: "abc" }, foo2: { bar2: "xyz" }```

In arrays, situation is a bit different, check out following example:
```["a", "b", c: { d: "e" }, 0]```

Function:
``` function foo(bar) { ... }```